crypto/err/err_all_legacy.c

```diff
@@ -43,13 +43,20 @@
 #include "crypto/x509err.h"
 #include "crypto/x509v3err.h"
 
+#ifdef OPENSSL_NO_ERR
+# define IMPLEMENT_LEGACY_ERR_LOAD(lib)         \
+    int ERR_load_##lib##_strings(void)          \
+    {                                           \
+        return 1;                               \
+    }
+#else
 # define IMPLEMENT_LEGACY_ERR_LOAD(lib)         \
     int ERR_load_##lib##_strings(void)          \
     {                                           \
         return err_load_##lib##_strings_int();  \
     }
+#endif
 
-# ifndef OPENSSL_NO_ERR
```

Comment:
However below this removed ifndef one space actually needs to be removed from preprocessor directives.